### DISCUSSION TITLE
Battery : 'low-at' option

### DISCUSSION THREAD

**Discussion by Crowerade** [polybar/polybar#2659] [Ideas]
Hi,
I have seen a bit of discussion over the ` low-at ` option in the battery module. I think the feature would be well welcomed to change directly the prefix or the entire display of the battery module when the battery is low. 
The setup would be very similar to the ` full-at ` option.

For some issues, I noticed that the usage of ` ramp ` was often proposed as an alternative solution to the changing display but I feel like the usage of ` ramp ` would be a lot wider than it should actually be.

**Comment by patrick96** [comment]
Do I have good news for you!

The newest version of polybar already supports that option: https://github.com/polybar/polybar/wiki/Module:-battery

**Reply by Crowerade** [reply]
Well, that was fast ! Thank you !
